tests/conftest.py

```diff
@@ -14,11 +18,27 @@ def __init__(self, app: Typer, test_snowcli_config: str):
 
     @functools.wraps(CliRunner.invoke)
     def invoke(self, *a, **kw):
+        return self.invoke_with_config_file(self.test_snowcli_config, *a, **kw)
```

Comment:
Previously:
- a lot of tests were using `invoke` which didn't use test.conf from resources so either local config file was used or config file loaded in some other test (incorrect behaviour)
- some tests were using `invoke_with_config` which used test.conf from resources (correct behaviour)

Now:
- `invoke` uses test.conf from resources by default
- Almost all the tests use `invoke`
- A few tests with custom config use new `invoke_with_config_file` function

Reply:
+ I moved `SnowCliRunner` to `fixtures.py` to resolve circular import